src/files/3_AllProjectsAnalyzers.editorconfig

```diff
@@ -393,13 +393,13 @@ dotnet_diagnostic.CA1200.severity = none
 # CA1310: Specify StringComparison for correctness (ex: string.Compare)
 # CA1311: Specify a culture or use an invariant version (ex: string.ToUpper)
 dotnet_diagnostic.CA1303.severity = none
-dotnet_diagnostic.CA1304.severity = suggestion
-dotnet_diagnostic.CA1305.severity = suggestion
-dotnet_diagnostic.CA1307.severity = suggestion
+dotnet_diagnostic.CA1304.severity = silent
+dotnet_diagnostic.CA1305.severity = silent
```

Comment:
These ones look important to avoid issues with implicit current culture. I wonder if it should be a warning.

Reply:
The change was reverted back suggestion. Changing it to a warning could be considered in the future